pages/edu/index.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <div>
+    <section class="header__partners">
+      <img
+        src="~/assets/images/partners.png"
+        alt="Financé par le Gouvernement de la République française, le plan France Relance et l'Union Européenne (NextGenerationEU)"
```

Comment:
Un retour qu'on avait eu dans l'audit d'a11y déjà :
```suggestion
        alt="Financé par le Gouvernement de la République française, liberté égalité fraternité, le plan France Relance et l'Union Européenne (NextGenerationEU)"
```